src/api/gen/api.ts

```diff
@@ -61,8 +50,7 @@ export class BaseAPI {
   }
 
   public getConfiguration = async () => {
```

Comment:
Il faut enlever le `async` (et les `await` dans les méthodes qui l'utilisent)
```suggestion
  public getConfiguration = () => {
```